src/openrct2/object/Object.h

```diff
@@ -366,6 +369,14 @@ class Object
 
     const std::vector<std::string>& GetAuthors() const;
     void SetAuthors(std::vector<std::string>&& authors);
+    Version& GetVersion()
```

Comment:
```suggestion
    const Version& GetVersion() const
```

This should either return a `const&` to prevent modifications of the member variable or a copy,